app/common/auth/authorisation_helper.py

```diff
@@ -0,0 +1,82 @@
+from uuid import UUID
+
+from app.common.data.models_user import User
+from app.common.data.types import GRANT_ROLES_MAPPING, RoleEnum
+
+
+class AuthorisationHelper:
+    @staticmethod
+    def has_logged_in(user: User) -> bool:
```

Comment:
question/idea: what do you think about all of these methods have a default user value of the current user?

Then for 99% of cases you don't need to pass the user in and can just do `authorisation_helper.is_platform_admin()`. On the rare occassion you want to check _someone else's_ permission you can pass it in explicitly, but I think nearly always you'll be checking the permissions of the person making the request.

It's a bit more magical and a bit less explicit, but a bit more concise. Pros and cons. Happy for you to decide 🙇